admin/database/database.go

```diff
@@ -129,6 +129,11 @@ type DB interface {
 	FindProjectInvite(ctx context.Context, projectID, userEmail string) (*ProjectInvite, error)
 	InsertProjectInvite(ctx context.Context, email, projectID, roleID, invitedByID string) error
 	DeleteProjectInvite(ctx context.Context, id string) error
+
+	CountOrganizationProjects(ctx context.Context, orgID string) (int, error)
+	CountOrganizationDeploymentsAndSlots(ctx context.Context, orgID string) (*OrganizationDeploymentsAndSlots, error)
+	CountOrganizationOutstandingInvitations(ctx context.Context, orgID string) (int, error)
+	CountSingleUserOrganizationForUser(ctx context.Context, userID string) (int, error)
```

Comment:
Also, consider grouping by object (project, deployment, invites, etc.) instead of verb (`Count`), like the other functions in this interface.